Svc/ActiveRateGroup/ActiveRateGroup.cpp

```diff
@@ -68,15 +68,19 @@ namespace Svc {
             }
         }
 
-        // grab timer for end of cycle
-        end.take();
+        // grab timer for endTime of cycle
+        endTime.now();
 
         // get rate group execution time
-        U32 cycle_time = end.diffUSec(cycleStart);
+        U32 cycleTime;
+        Os::RawTime::Status status = endTime.getDiffUsec(cycleStart, cycleTime);
+        if (status != Os::RawTime::OP_OK) {
```

Comment:
Don't need this if() clause since we are going to do it in getDiffUsec (review comment above)